openedx/features/edly/tests/test_models.py

```diff
@@ -0,0 +1,102 @@
+"""
+Tests for Edly django models.
+"""
+
+from django.test import TestCase
+from openedx.features.edly.models import EdlyOrganization, EdlySubOrganization
+from openedx.features.edly.tests.factories import EdlyOrganizationFactory, SiteFactory, EdlySubOrganizationFactory
+
+
+class OrganizationTests(TestCase):
+    """
+    Tests for Organization Model.
+    """
+
+    @classmethod
+    def setUpClass(cls):
+        super(OrganizationTests, cls).setUpClass()
+
+    def test_organization_post_save_receiver(self):
+        """
+        Test entry is added to organization model and filter on it's value.
+        """
+        # add Organization entry to database
+        org_name = 'test org name'
+        organization = EdlyOrganizationFactory(name=org_name)
+
+        # Verify an entry to Organization was added.
+        organizations = EdlyOrganization.objects.filter(name=org_name).all()
+
+        # Make sure an entry (and only one entry) is saved for Organization
+        assert len(organizations) == 1
+
+    def test_site_configuration_post_update_receiver(self):
+        """
+        Test that entry is updated in Organization model.
+        """
+
+        org_name = 'updated test org name'
+
+        # add SiteConfiguration to database
+        organization = EdlyOrganizationFactory()
+
+        organization.name = org_name
+        organization.save()
+
+        # Verify an entry to Organization was updated.
+        organizations = EdlyOrganization.objects.filter(name=org_name).all()
+
+        # Verify an entry to Organization was updated is filtered
+        assert organizations[0].name == org_name
+        assert len(organizations) == 1
```

Comment:
Assert the length for list first before asserting its attributes

Reply:
Done